src/GslibIO.jl

```diff
@@ -52,28 +92,20 @@ Optionally set the value used for missings `na`.
 """
 function load_legacy(filename::AbstractString, dims::NTuple{3,Int};
                      origin=(0.,0.,0.), spacing=(1.,1.,1.), na=-999)
-  open(filename) do fs
-    # skip header
-    readline(fs)
 
-    # number of properties
-    nvars = parse.(Int, readline(fs))
+  # parse the GSLIB file
+  X, metadata = parse_gslib(filename)                     
```

Comment:
Please choose a single identifier and stick to it. Here you are renaming `data` to `X`. The reviewer of your contribution has to maintain two different identifiers in his/her head.

Reply:
I did not quite understand you comment. The current code already has a data variable used in line 99 and was using the X variable. That is the reason to keep X as variable.

Reply:
You can rename this `X` variable to `data` and use the same identifier inside the `parse_legacy` function.

Reply:
But then the variable `data` will be replaced by something else (not good).

Reply:
I will use datadict for the variable name after `# create data dictionary`

Reply:
Please use `data` as suggested, the identifier `X` isn't very good in this context. Also, to be consistent with the current view of the project, we look at these as tables as opposed to dictionaries. You can use either `data` or `table` for this identifier. It should be clear from the context.